app/models/cost_item.rb

```diff
@@ -1,5 +1,12 @@
 class CostItem < ActiveRecord::Base
   has_many :costs
 
+  validates :web_cost, inclusion: { in: %w(yes no) }
+  validates :name, :cost_group, presence: true
+
   scope :current, -> { where(current: true) }
+
+  def self.delivery_partners
+    Partners.delivery_partners
```

Comment:
I've renamed this method `allowed_delivery_partners` which is a better description of it's purpose